library/psa_crypto.c

```diff
@@ -1421,6 +1421,67 @@ psa_status_t psa_hash_verify( psa_hash_operation_t *operation,
     return( PSA_SUCCESS );
 }
 
+psa_status_t psa_hash_clone(const psa_hash_operation_t *source_operation,
```

Comment:
Usually, the context acted upon is listed as the first argument; I'd therefore prefer the destination context to come first.

Reply:
There's no such convention in this API. There's a convention that input comes before output, and a convention that the context comes before other parameters. With an input context and an output context, the input context comes first.

Reply:
Ok.